README.md

```diff
@@ -1,8 +1,18 @@
 # examples
-Collection of example OpenRPC service definition files
```

Comment:
should we leave this at the top to help npmjs.org descriptions?

Reply:
this should be done in the package.json though

Reply:
https://github.com/open-rpc/examples/pull/147